refactor: Remove deprecated xwf functionality

alexzurbonsen opened the pull request:
## Summary

Relates to https://github.com/magma/magma/issues/13546. This PR removes xwf references from scripts. 

`deploy.sh` is deleted entirely since it seems to be unused and there are no references to it in the documentation.

## Test Plan

- CI
- Test install_gateway.sh individually?

## Additional Information

- [ ] This change is backwards-breaking

<!--
    If this is a backwards-breaking change, document the upgrade instructions.
    All upgrade instructions for backwards-breaking changes will be aggregated
    in the next release's changelog so this is very important to fill out.
-->

alexzurbonsen committed:
remove xwf references in scripts

Signed-off-by: Alexander zur Bonsen <alexander.zur.bonsen@tngtech.com>

crasu reviewed (commented):
Could  you split up this pr into one for the k8s removal and one for the pipelined removal?

alexzurbonsen committed:
remove deploy.sh and config template for xwfv3

feg/radius deploy.sh is not used by any code or referenced in any
documentation and thus considered safe to delete.

The config template for xwfv3 has become superfluous since xwf has been deleted.

Signed-off-by: Alexander zur Bonsen <alexander.zur.bonsen@tngtech.com>
